src/js/rtp-stats.js

```diff
@@ -62,42 +61,32 @@ export function extractMediaStatsFromStats(timestamp, stats, streamType) {
-            } else if (statsReport.type === 'outbound-rtp' && !statsReport.isRemote && (streamType === 'audio_output' || streamType === 'video_output')) {
+                // Standardized report for packets sent
+            } else if (statsReport.type === 'outbound-rtp' && streamType === 'audio_output') {
                 // outbound-rtp report can appear either before or after extractedStats object is created
-                if(extractedStats && !extractedStats.packetsCount) {
-                    extractedStats.packetsCount = statsReport.packetsSent;
-                } else {
-                    packetsSent = statsReport.packetsSent;
-                }
-            // Standardized report for remaining output stream stats
-            } else if (statsReport.type === 'remote-inbound-rtp' && (streamType === 'audio_output' || streamType === 'video_output')) {
-                reportType = statsReport.type;
-                extractedStats = {
-                    timestamp:          timestamp,
-                    packetsLost:        statsReport.packetsLost,
-                    packetsCount:       packetsSent,
-                    rttMilliseconds:    Number.isInteger(statsReport.roundTripTime) ? statsReport.roundTripTime : is_defined(statsReport.roundTripTime) ? statsReport.roundTripTime * 1000 : null,
-                    jbMilliseconds:     when_defined(statsReport.jitter, 0) * 1000
-                };
-            // Case for Firefox 65 and below for getting remaining output stream stats
-            } else if (statsReport.type === 'inbound-rtp' && statsReport.isRemote && (streamType === 'audio_output' || streamType === 'video_output')) {
+                extractedStats = extractedStats || {};
+                extractedStats.packetsCount = statsReport.packetsSent;
+                // Standardized report for output audio level
+            } else if (statsReport.type === 'media-source' && streamType === 'audio_output') {
+                extractedStats = extractedStats || {};
+                extractedStats.audioLevel = is_defined(statsReport.audioLevel) ? Math.floor(statsReport.audioLevel * 32768) : null;
+                // Standardized report for remaining output stream stats
+            } else if (statsReport.type === 'remote-inbound-rtp' && streamType === 'audio_output') {
                 reportType = statsReport.type;
-                extractedStats = {
-                    timestamp:          timestamp,
-                    packetsLost:        statsReport.packetsLost,
-                    packetsCount:       packetsSent,
-                    rttMilliseconds:    Number.isInteger(statsReport.roundTripTime) ? statsReport.roundTripTime : is_defined(statsReport.roundTripTime) ? statsReport.roundTripTime * 1000 : null,
-                    jbMilliseconds:     when_defined(statsReport.jitter, 0) * 1000
-                };
+                extractedStats = extractedStats || {};
+                extractedStats.timestamp=timestamp;
```

Comment:
Nit: Space after and before =
Applies to below 3 lines and rest of the places

Reply:
will address in the next revision, thanks!